edx_proctoring/settings/common.py

```diff
@@ -0,0 +1,38 @@
+"Common Pluggable Django App settings"
+
+
+def plugin_settings(settings):
+    "Injects local settings into django settings"
+    settings.PROCTORING_SETTINGS = {}
+    settings.PROCTORING_BACKENDS = {
+        'DEFAULT': 'null',
+        'null': {},
+    }
+    proctoring_js = (
+        [
+            'proctoring/js/models/proctored_exam_allowance_model.js',
+            'proctoring/js/models/proctored_exam_attempt_model.js',
+            'proctoring/js/models/proctored_exam_model.js'
+        ] +
+        [
```

Comment:
Why use this [..] + [..] syntax?

Reply:
Matt points out this is the way the lines were in the settings file.  We don't need to keep them like this.

Reply:
:+1: